<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmEditCunas</class>
 <widget class="QDialog" name="FrmEditCunas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cuñas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 48, 58);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="BtnModificar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Modificar cuña</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(32, 39, 47);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
        </property>
        <property name="icon">
         <iconset resource="radit.qrc">
          <normaloff>:/general/iconos/menu/añadir.png</normaloff>:/general/iconos/menu/añadir.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnEliminar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Eliminar cuña</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

  background-color: rgb(32, 39, 47);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
        </property>
        <property name="icon">
         <iconset resource="radit.qrc">
          <normaloff>:/general/iconos/menu/delete.png</normaloff>:/general/iconos/menu/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCue">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Cue</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

  background-color: rgb(32, 39, 47);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
        </property>
        <property name="icon">
         <iconset resource="radit.qrc">
          <normaloff>:/general/iconos/player/play.png</normaloff>:/general/iconos/player/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnStop">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

  background-color: rgb(32, 39, 47);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
        </property>
        <property name="icon">
         <iconset resource="radit.qrc">
          <normaloff>:/general/iconos/player/stop.png</normaloff>:/general/iconos/player/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(33, 48, 58);
 
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>Página:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboPagina">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnRenombrar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
        </property>
        <property name="text">
         <string>Renombrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="Lista">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
background-color: rgb(32, 39, 47);

	
	color: rgb(7, 250, 242);
	
	gridline-color: rgb(117, 125, 164);

}

QHeaderView::section
{
spacing: 10px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #5aa3fd, stop: 1 #1f406a);
color: white;
border: 1px solid black;
margin: 1px;
text-align: right;
font-family: arial;
font-size:12px;
}
</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>26</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>22</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>16</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Cuñas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Url</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n1.png</normaloff>:/numeros/iconos/numeros/n1.png</iconset>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n2.png</normaloff>:/numeros/iconos/numeros/n2.png</iconset>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n3.png</normaloff>:/numeros/iconos/numeros/n3.png</iconset>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n4.png</normaloff>:/numeros/iconos/numeros/n4.png</iconset>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n5.png</normaloff>:/numeros/iconos/numeros/n5.png</iconset>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n6.png</normaloff>:/numeros/iconos/numeros/n6.png</iconset>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n7.png</normaloff>:/numeros/iconos/numeros/n7.png</iconset>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n8.png</normaloff>:/numeros/iconos/numeros/n8.png</iconset>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="7" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="radit.qrc">
         <normaloff>:/numeros/iconos/numeros/n9.png</normaloff>:/numeros/iconos/numeros/n9.png</iconset>
       </property>
      </item>
      <item row="8" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="8" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="BtnAplicar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
      </property>
      <property name="text">
       <string>Aplicar...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnAceptar">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="radit.qrc"/>
 </resources>
 <connections/>
</ui>
